db/tikv/txn.go

```diff
@@ -153,43 +154,43 @@ func (db *txnDB) Scan(ctx context.Context, table string, startKey string, count
 }
 
 func (db *txnDB) Update(ctx context.Context, table string, key string, values map[string][]byte) error {
-	rowKey := db.getRowKey(table, key)
+	// Suppress RunInNewTxn warning log.
+	log.SetLevel(log.ErrorLevel)
```

Comment:
This is to suppress error printed from the tidb/kv package, but I think it should move to a better place and outside of this PR.

Reply:
yes, I think we don't need logging. so you don't need to import log module.

Reply:
or if you want to suppress these logs, I think we should set level at first or in `init` function, not in update function.